Update providers.py

techux opened the pull request:
added more providers on base_providers list
Get some extra providers from https://www.dnsbl.info/dnsbl-database-check.php

techux committed:
Update providers.py

added more providers on base_providers list

dmippolitov commented:
Hi, thanks for your contribution - there is at least one DNSBL provider with false positives on 8.8.8.8 according to tests, could you please check again the list and remove it?

techux commented:
Okay , i will check it once again

On Sun, May 28, 2023 at 2:36 AM Dmitry ***@***.***> wrote:

> Hi, thanks for your contribution - there is at least one DNSBL provider
> with false positives on 8.8.8.8 according to tests, could you please check
> again the list?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dmippolitov/pydnsbl/pull/30#issuecomment-1565683431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN3KLKICON4ODGV3LZMIRETXIJUHBANCNFSM6AAAAAAYRGMLGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

techux committed:
Update providers.py

removed dul.ru

techux committed:
Update providers.py

revert base providers to original list

techux committed:
Update providers.py

added some providers

web.dnsbl.sorbs.net
sbl.spamhaus.org
spam.abuse.ch
dnsbl.dronebl.org
ubl.unsubscore.com
noptr.spamrats.com
